From akpm@osdl.org  Wed Dec 20 11:20:11 2006
Message-Id: <200612192115.kBJLFQj8010844@shell0.pdx.osdl.net>
From: Olivier Galibert <galibert@pobox.com>
Subject: bluetooth: add support for another Kensington dongle
To: greg@kroah.com
Cc: akpm@osdl.org, galibert@pobox.com, marcel@holtmann.org
Date: Tue, 19 Dec 2006 13:15:25 -0800

From: Olivier Galibert <galibert@pobox.com>

Add the stupid sco fixup quirk to yet another Broadcom/Kensington device.

Cc: Marcel Holtmann <marcel@holtmann.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/bluetooth/hci_usb.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/bluetooth/hci_usb.c
+++ gregkh-2.6/drivers/bluetooth/hci_usb.c
@@ -126,6 +126,7 @@ static struct usb_device_id blacklist_id
 
 	/* Kensington Bluetooth USB adapter */
 	{ USB_DEVICE(0x047d, 0x105d), .driver_info = HCI_RESET },
+	{ USB_DEVICE(0x047d, 0x105e), .driver_info = HCI_WRONG_SCO_MTU },
 
 	/* ISSC Bluetooth Adapter v3.1 */
 	{ USB_DEVICE(0x1131, 0x1001), .driver_info = HCI_RESET },
